pwndbg/gdblib/heap/ptmalloc.py

```diff
@@ -758,8 +758,7 @@ def fastbins(self) -> Bins:
         fd_offset = pwndbg.gdblib.arch.ptrsize * 2
         safe_lnk = pwndbg.glibc.check_safe_linking()
         result = Bins(BinType.FAST)
-        num_fastbins = 7
-        for i in range(num_fastbins):
+        for i in range(NFASTBINS):
```

Comment:
if NFASTBINS=10 why was this 7 before in the first place? @CptGibbon

Reply:
Because the array for fastbins in glibc is 10 pointers long, but only seven are ever used.